crates/egui/src/containers/window.rs

```diff
@@ -390,7 +390,11 @@ impl<'open> Window<'open> {
             with_title_bar,
         } = self;
 
+        let is_frame_overrode = frame.is_some();
         let frame = frame.unwrap_or_else(|| Frame::window(&ctx.style()));
+        let frame_fill = frame.fill;
+        let frame_margin = frame.inner_margin;
+        let frame_rounding = frame.rounding;
```

Comment:
this feels like unnecessary pollution of the namespace. Why not use `frame.fill` later?

Reply:
OK, it has been modified, please check again.